contents/docs/session-replay/sharing.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Sharing or embedding recordings
+---
+
+Recordings are typically consumed from the **Session Replay** part of PostHog but in some cases you may want to share specific recordings with people outside of your PostHog organization. 
+
+
+<iframe allowfullscreen width="100%" height="450" frameborder="0" src="https://app.posthog.com/embedded/VDVn0WRlvJdBomoFyy2Xg0Di2T1pEg"></iframe>
+
+> Above is an embedded recording of posthog.com ... on posthog.com  - very meta.
```

Comment:
I hate to be the fun police, but I dont think this adds value :) 
```suggestion
```

Reply:
I also disagree - we are all about whimsy and fun at PostHog right? Why would we _not_ want this?

Reply:
I also would really like this as a dogfooding mechanism. Given we likely won't use this feature much ourselves, it's good to have an easy place to reference that it is definitely working